modules/openapi-generator/src/main/resources/swift5/Extensions.mustache

```diff
@@ -192,4 +194,206 @@ extension RequestBuilder {
         }
         return deferred.promise
     }
-}{{/usePromiseKit}}
+}{{/usePromiseKit}}{{#useVapor}}
+
+public enum JSON: Content {
```

Comment:
@aymanbagabas What was the reference implementation of this?

Reply:
The JSON type is based on https://github.com/vapor/vapor/commit/b7786e8a60c71c2e86765eacd9f403f076a081b8 by @tanner0101. I'll add that to the commit description.

Reply:
We just need to be sure that there is no licensing issue here.